crates/host/src/wasmbus/mod.rs

```diff
@@ -181,20 +188,24 @@ impl Queue {
         host_key: &KeyPair,
     ) -> anyhow::Result<Self> {
         let host_id = host_key.public_key();
-        let (registries, pings, links, queries, auction, commands, inventory) = try_join!(
+        let (registries, pings, links, queries, auction, commands, inventory, config) = try_join!(
```

Comment:
Since we're considering adding several topics to the control interface, should we put this functionality behind an `--experimental` flag on the host? Separate/fine-grained feature flags would be better, but we don't have a way to do that yet

Reply:
I don't really like the idea of needing to specify a flag to turn this on unless we truly think this is too experimental for people to use normally. We really should have some versioned APIs instead, but that would require changing all topics

Reply:
Synced offline and agreed this one doesn't need an experimental flag